src/cli.ts

```diff
@@ -349,7 +351,28 @@ async function codegen(options: Options, url: string | undefined, target: string
     default: throw new Error(`Invalid target: '${target}'`);
   }
 
-  const { context, browserName, launchOptions, contextOptions } = await launchContext(options, false);
+  let context: BrowserContext;
+  let browserName: string;
+  let launchOptions: playwright.LaunchOptions;
+  let contextOptions: playwright.BrowserContextOptions
+  if(electronPath)
+  {  
+    const app = await electron.launch(electronPath, {});
+    context = app.context() as any as BrowserContext;
+    browserName = 'electron';
+    launchOptions = {};
+    contextOptions = {};
+    // TODO: register some sort of close handler to exit the script when electron exits. for some reason page close isn't firing.
+
+    // for some reason, we need to launch something else in order to get left mouse clicks to reach our application. wtf?
+    await playwright.chromium.launch();
```

Comment:
That's strange :)

Reply:
This was just adding a small delay. The real problem was a simple race condition - if playwright-cli's RecorderController tried to call exposeBinding before the browser process window was opened, then none of the the bindings would get installed in the renderer, but the script would still get injected, which would throw errors in every click handler.

I just added a fixed that awaits the first window to open before constructing the ScriptController.